x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -226,7 +226,7 @@ export const HostList = () => {
             >
               <FormattedMessage
                 id="xpack.securitySolution.endpointList.hostStatusValue"
-                defaultMessage="{hostStatus, select, online {Online} error {Error} other {Offline}}"
+                defaultMessage="{hostStatus, select, online {Online} error {Error} unenrolling {Unenrolling} other {Offline}}"
```

Comment:
Do we have any front-end test cases for this status? if so, and it attempt to validate possible values, should a `Unenrolling` test be added.

Reply:
I've added the status to tests, not sure if it tests any labels though.